src/data/repositories/consts/DiseaseOutbreakConstants.ts

```diff
@@ -19,6 +24,7 @@ export const hazardTypeCodeMap: Record<HazardType, string> = {
 
 export const diseaseOutbreakCodes = {
     name: "RTSL_ZEB_TEA_EVENT_NAME",
+    dataSource: "RTSL_ZEB_TEA_DATA_SOURCE",
```

Comment:
please merge with Ana's feature/add-data-source-create-event branch and change the PR to merge into feature/add-data-source-create-event branch instead , so this change will not show up as a merge conflict